quinn/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "quinn"
-version = "0.11.0"
```

Comment:
Should we skip to 0.11.2 for consistency/less confusing git tags?

Reply:
Skipping versions just feels differently confusing. I think we already have plenty of tags that are confusing in exactly the same way?